go.mod

```diff
@@ -44,6 +44,7 @@ require (
 	github.com/go-redis/redis v6.15.2+incompatible
 	github.com/go-sql-driver/mysql v1.4.1
 	github.com/go-swagger/go-swagger v0.20.1
+	github.com/go-xorm/xorm v0.7.9
```

Comment:
Currently we will use `xorm.io/xorm v0.8.0`.

Reply:
Will update.

Reply:
Fixed